app/models/concerns/sirsi_response.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class SirsiResponse
+  attr_reader :response_raw, :response_hash
+
+  def initialize(response)
+    @response_raw = response.body
```

Comment:
we only need this for error reporting, right?

Reply:
Like the whole class or just `@response_raw`?

If the latter, I'm only calling it at line 21 of `place_holds_job.rb`

```rb
        Sidekiq.logger.error("place_holds_results_#{patron_key} #{barcode}: #{sirsi_response.response_raw}")
```

You thinking we should get rid of it?

Reply:
was asking for `@response_raw`, yeah I just found at `place_holds_job.rb` as well, no we should not get rid of it, I was just trying to understand :)

Reply:
I wouldn't be opposed :)